src/styles/theme.ts

```diff
@@ -0,0 +1,26 @@
+import '@fontsource/space-grotesk/500.css';
+import '@fontsource/space-grotesk/700.css';
+
```

Comment:
duplicated font files. you can do something like this https://github.com/chakra-ui/chakra-ui/discussions/7235#discussioncomment-4685189